lib/Controller/ConfigController.php

```diff
@@ -12,53 +12,38 @@
 namespace OCA\Jira\Controller;
 
 use DateTime;
-use OCP\IURLGenerator;
+use OCA\Jira\AppInfo\Application;
+use OCA\Jira\Service\NetworkService;
+use OCP\AppFramework\Controller;
+use OCP\AppFramework\Http\DataResponse;
+use OCP\AppFramework\Http\RedirectResponse;
 use OCP\IConfig;
 use OCP\IL10N;
-use OCP\AppFramework\Http\RedirectResponse;
-use OCP\IRequest;
-use OCP\AppFramework\Http\DataResponse;
-use OCP\AppFramework\Controller;
 
-use OCA\Jira\Service\JiraAPIService;
-use OCA\Jira\AppInfo\Application;
+use OCP\IRequest;
+use OCP\IURLGenerator;
 
 class ConfigController extends Controller {
 
-	/**
-	 * @var IConfig
-	 */
-	private $config;
-	/**
-	 * @var IURLGenerator
-	 */
-	private $urlGenerator;
-	/**
-	 * @var IL10N
-	 */
-	private $l;
-	/**
-	 * @var JiraAPIService
-	 */
-	private $jiraAPIService;
-	/**
-	 * @var string|null
-	 */
-	private $userId;
+	private IConfig $config;
+	private IURLGenerator $urlGenerator;
+	private IL10N $l;
+	private ?string $userId;
+	private NetworkService $networkService;
 
 	public function __construct(string $appName,
-								IRequest $request,
-								IConfig $config,
-								IURLGenerator $urlGenerator,
-								IL10N $l,
-								JiraAPIService $jiraAPIService,
-								?string $userId) {
+		IRequest $request,
+		IConfig $config,
+		IURLGenerator $urlGenerator,
+		IL10N $l,
+		NetworkService $networkService,
+		?string $userId) {
```

Comment:
```suggestion
		IRequest $request,
		private IConfig $config,
		private IURLGenerator $urlGenerator,
		private IL10N $l,
		private NetworkService $networkService,
		private ?string $userId) {
```
(Just in case) In Php >= 8.0, if you do this you can get rid of the property declaration (before the constructor) and of the property assignment (in the constructor).

Reply:
Yeah, I know but I like the old style for some reason more..